### PR TITLE
Add option to escape HTML instead of clobber (fix #149)

### LINKED ISSUE #149: Escape HTML instead of clobbering?
Does/will comrak support escaping HTML, so it displays in a browser as typed, rather than clobbering?

### PR DESCRIPTION
**Pull request by yujiri8**
This is my first time contributing to a Rust library :)

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -134,6 +134,7 @@ fn main() -> Result<(), Box<dyn Error>> {
                 .parse()
                 .unwrap_or(0),
             unsafe_: matches.is_present("unsafe"),
+            escape: matches.is_present("escape"),
```
Comment (kivikakk) [inline src/main.rs:137]: We'll need to add a corresponding `clap::Arg` above to get this option from the command line.

Comment (yujiri8) [reply src/main.rs:142]: I've updated the commit to include command-line support.

**Comment by kivikakk** [comment]
Looking good! Will merge and prepare a new release.

**Comment by kivikakk** [comment]
The release is done! Your changes are now part of [comrak 0.8.1](https://crates.io/crates/comrak). Thank you so much! :heart:

**Pull request merged**